lib/utils/blockString.js

```diff
@@ -10,15 +10,14 @@ const rawNodeString = require('./rawNodeString');
 /**
  * Return a CSS statement's block -- the string that starts and `{` and ends with `}`.
  *
- * If the statement has no block (e.g. `@import url(foo.css);`),
- * return false.
+ * If the statement has no block (e.g. `@import url(foo.css);`), returns an empty string.
  *
  * @param {Rule | AtRule} statement - postcss rule or at-rule node
- * @return {string | boolean}
+ * @returns {string}
  */
-module.exports = function (statement) {
+module.exports = function blockString(statement) {
 	if (!hasBlock(statement)) {
-		return false;
+		return '';
```

Comment:
[note] I don't know why this function has returned a boolean, but it seems to make sense that it returns a string instead.

For example, it appears returning a boolean is incorrect at the following place, but errors never occur.
Perhaps, it has no problems because of calling `hasBlock()` before `blockString()`.

https://github.com/stylelint/stylelint/blob/403248c5586d6015127b5f98cf93442efc7c2547/lib/rules/block-opening-brace-space-after/index.js#L51

https://github.com/stylelint/stylelint/blob/403248c5586d6015127b5f98cf93442efc7c2547/lib/rules/block-opening-brace-space-after/index.js#L56